client/connector_create.go

```diff
@@ -50,6 +61,17 @@ func (cli *VanClient) ConnectorCreateFromFile(ctx context.Context, secretFile st
 	if options.Name == "" {
 		options.Name = secret.ObjectMeta.Name
 	}
+	ownToken, err := cli.isOwnToken(ctx, secret)
+	if err != nil {
+		return nil, fmt.Errorf("Can't check secret ownership: |%s|", err.Error())
+	}
+	if ownToken {
+		err := kube.DeleteSecret(options.Name, options.SkupperNamespace, cli.KubeClient)
+		if err != nil {
+			return nil, fmt.Errorf("Created connection to self with token |%s|! Error attempting to remove it: |%s|", secretFile, err.Error())
```

Comment:
A vertical line common as a separator, would single quote be a better option?

Reply:
Yes, good point.